src/MoBi.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -105,12 +112,13 @@ private async Task<ModelProject> mapToModel(SnapshotProject projectSnapshot, Pro
          NumberOfSimulationsLoaded = 0
       };
 
+      List<Task<MoBiSimulation>> tasks = [];
       foreach (var simulationSnapshot in projectSnapshot.Simulations)
       {
          try
          {
-            var simulation = await _simulationMapper.MapToModel(simulationSnapshot, simulationContext);
-            addSimulations(project, simulation);
+            tasks.Add(_simulationMapper.MapToModel(simulationSnapshot, simulationContext));
+
```

Comment:
I think this should revert to the original code. We don't want to run the construction of simulations in parallel, only the actual simulation run.